test/train-sets/ref/multilabel_ldf.stderr

```diff
@@ -25,4 +25,4 @@ number of examples = 8
 weighted example sum = 8.000000
 weighted label sum = 0.000000
 average loss = 0.875000
-total feature number = 24
+total feature number = 66
```

Comment:
do we understand why this changed?

Reply:
It was treating each example separately despite being in a multiex and the featuer counting was off as a result